### ISSUE TITLE
linux: shared object file not found

### ISSUE THREAD

**Issue by simar7** [fyne-io/fyne-cross#130]
### Describe the bug:
```
$ fyne-cross linux
$ ./fyne-cross/bin/linux-amd64/fyne-tracee-events 
$ ./fyne-cross/bin/linux-amd64/fyne-tracee-events: error while loading shared libraries: libGL.so.1: cannot open shared object file: No such file or directory
```

### To Reproduce:
Steps to reproduce the behaviour:
1. Install fyne-cross via go get
2. Build linux package on linux, via fyne-cross linux
3. See error

### Example code:
```
package main
import "fmt"
func main(){
fmt.Println("hello world")
}
```

### Device and debug info (please complete the following information):

```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 21.10
Release:        21.10
Codename:       impish
```

**Comment by simar7** [comment]
The same issue also occurs when cross compiling on `darwin` for `linux`.

**Comment by andydotxyz** [comment]
I don't know if it is related or not, but fyne-cross is designed to cross-compile Fyne apps. The demo code you pasted isn't a fyne app...

**Comment by simar7** [comment]
> I don't know if it is related or not, but fyne-cross is designed to cross-compile Fyne apps. The demo code you pasted isn't a fyne app...

Yes apologies for the placeholder text. I've updated it to the right repo.

**Comment by lucor** [comment]
Could you post the output of `ldd fyne-cross/bin/linux-amd64/fyne-tracee-events` ?
It looks like the library `libGL` is missed... could by any chance the X server not installed on your linux box?
